index.rst

```diff
@@ -46,6 +46,7 @@ Systems
    :maxdepth: 2
 
    systems/index
+   ace_testbed/index
```

Comment:
this one and the index item in line 106 below is causing the link to the ACE testbed doc to appear twice on the sidebar. There should only be one (though I don't know which one would be better). My vote is to keep the one on line 106 and delete this one.

Reply:
Forgot to delete this one at line 49 after adding line 106. Removing line 49.